### PR TITLE
Military Overhaul: the Artifacts

### PR DESCRIPTION
**Pull request by zumbak04**
Changelog:
- Rebalanced combat modifiers of vanilla artifacts to make sure all units receive the same amount of bonuses in total. For example, crafted swords, lances, axes, maces, bows give (in total) 80% bonus to Casters (vanilla Light Infantry), 80% to Ranged Infantry (vanilla Archers), 110% to Infantry (vanilla Heavy Infantry), 110% to Large Infantry (vanilla Pikemen), 110% to Cavalry (vanilla Light Cavalry) and 60% to Aerial Infantry (vanilla Knights).
- Aqir, Arakkoa, Centaur, Thalassian, Darnassian, Nazja, Sethrak, Troll, Vulpera culture groups and Blackwater, Kul Tiran, Southern, Wastewander cultures and Tidemother and Tidehunter religions can create scimitars.
- Event that happens after you invite weaponsmith where you select type of your future weapon mentions correct combat modifiers (like Casters, Large Infantry and so on).

### CODE REVIEW

File: localisation/00_zuwc_united_localisation.csv [hunk 1]
```diff
@@ -3000,12 +3000,12 @@ EVTOPTB_WOL_11332;I would rather pursue other disciplines...;;;;;;;;;;;;;x
...
-EVTOPTF_MNM_9118_TT;Bows may give a bonus to commanded ?Ranged Infantry?! and ?YCavalry?!, as well as ?Ymartial?!, ?YPersonal Combat Skill?! and ?YMonthly Prestige?!.\n;;;;;;;;;;;;;x
+EVTOPTA_MNM_9118_TT;Swords may give a bonus to commanded �Y[GetActualLightInfantryName]�! and �Y[GetHeavyInfantryName]�!, as well as �YMartial�!, �YPersonal Combat Skill�! and �YMonthly Prestige�!.\n;;;;;;;;;;;;;x
+EVTOPTB_MNM_9118_TT;Scimitars may give a bonus to commanded �Y[GetActualLightInfantryName]�! and �Y[GetHeavyInfantryName]�!, as well as �YMartial�!, �YPersonal Combat Skill�! and �YMonthly Prestige�!.\n;;;;;;;;;;;;;x
+EVTOPTC_MNM_9118_TT;Lances may give a bonus to commanded �Y[GetActualHeavyInfantryName]�! and �Y[GetCavalryName]�!, as well as �YMartial�!, �YPersonal Combat Skill�! and �YMonthly Prestige�!.\n;;;;;;;;;;;;;x
+EVTOPTD_MNM_9118_TT;Axes may give a bonus to commanded �Y[GetPikemenName]�!, as well as �Y[GetLightInfantryName]�!, as well as �YMartial�!, �YPersonal Combat Skill�! and �YMonthly Prestige�!.\n;;;;;;;;;;;;;x
```
Comment (MemeWizard) [inline localisation/00_zuwc_united_localisation.csv]: as well as well

**Comment by zumbak04** [comment]
> Also, imo, it's weird choice for modifier/weapon type
> Like, sword giving bonus to casters, but not infantry?

It gives bonus to Melee Infantry starting from 2 tier. Among bows, maces, axes and lancers, swords are the most suitable for casters.

File: localisation/000_wc_military_localization.csv [hunk 2]
```diff
@@ -51,14 +51,14 @@ HEAVY_INFANTRY_OFFENSIVE_FIRE;Infantry Attack;;Infanterie 
 HEAVY_INFANTRY_DEFENSIVE_FIRE;Infantry Defense;;Infanterie � Verteidigung;;;;;;;;;;;x
 HEAVY_INFANTRY_MORALE;Infantry Morale;;Infanterie � Moral;;;;;;;;;;;x
 HEAVY_INFANTRY_MODIFIER;Infantry Modifier;;Infanterie � Disziplin;;;;;;;;;;;x
-CU_HEAVY_INFANTRY;Melee Infantry;;Nahkampfinfanterie;;;;;;;;;;;x
+CU_HEAVY_INFANTRY;Melee Troops;;Nahkampfinfanterie;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/000_wc_military_localization.csv]: reeeee

File: localisation/000_wc_military_localization.csv [hunk 3]
```diff
@@ -51,14 +51,14 @@ HEAVY_INFANTRY_OFFENSIVE_FIRE;Infantry Attack;;Infanterie 
...
 CU_ACTUAL_HEAVY_INFANTRY;Infantry;;Infanterie;;;;;;;;;;;x
 LIGHT_INFANTRY;Casters;;Zauberer;;;;;;;;;;;x
 LIGHT_INFANTRY_OFFENSIVE_FIRE;Casters Attack;;Zauberer � Angriff;;;;;;;;;;;x
 LIGHT_INFANTRY_DEFENSIVE_FIRE;Casters Defense;;Zauberer � Verteidigung;;;;;;;;;;;x
 LIGHT_INFANTRY_MORALE;Casters Morale;;Zauberer � Moral;;;;;;;;;;;x
 LIGHT_INFANTRY_MODIFIER;Casters Modifier;;Zauberer � Disziplin;;;;;;;;;;;x
-CU_LIGHT_INFANTRY;Support Infantry;;Unterst�tzende Infanterie;;;;;;;;;;;x
+CU_LIGHT_INFANTRY;Support Troops;;Unterst�tzende Infanterie;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/000_wc_military_localization.csv]: reeeeee

File: localisation/000_wc_military_localization.csv (hunk 2 above)
Comment (zumbak04) [reply localisation/000_wc_military_localization.csv]: Why you're reeing? Unit groups called troops. That's the rule. Heavy troops, light troops, mounted troops.

Comment (DukeJeans) [reply localisation/000_wc_military_localization.csv]: I was just reeing for kicks. I agree with this change and all others in this pull request

**Pull request merged**